DevAcademy/Scenes/Places/PlacesScene.swift

```diff
@@ -43,7 +43,7 @@ struct PlacesScene: View {
                 } // TOOLBAR ITEM
             } // TOOLBAR
         } // NAVIGATION
-        .onAppear(perform: state.fetch)
+        .task{await state.fetch()}
```

Comment:
kdyz uz se tam nepredava ta funkce, ale je tam primo to jeji volani tak je podle me o neco citelnejsi zvolit tu variantu na vic radku.
```Swift
.task {
    await state.fetch()
}
```
Ale zalezi na osobni preferenci 😉